manifest-production.yml

```diff
@@ -1,13 +1,9 @@
 ---
-
 path: _site/
+instances: 3
```

Comment:
Wouldn't 2 be enough? I think the main reason for > 1 is so we dont have an outage if we lose a diego-cell?

Reply:
Probably 2 is fine.